srv/services.js

```diff
@@ -28,7 +28,32 @@ class ProcessorService extends cds.ApplicationService {
   }
 }
 
-module.exports = { ProcessorService }
+class AdminService extends cds.ApplicationService {
```

Comment:
as we're not serving service `S4`, we can't implement it either (i.e., `class S4 extends ...`)
→ implement in `AdminService` instead, which is somewhat a hack